README.md

```diff
@@ -344,6 +344,10 @@ participants:
     # - teku: consensys/teku:latest
     vc_image: ""
 
+    # The number of validator clients to run for this participant
```

Comment:
is this PR changing the # of validators or the # of validator clients for a participant?